keras_nlp/src/models/gemma/gemma_backbone_test.py

```diff
@@ -132,7 +132,40 @@ def test_distribution(self):
                 self.assertEqual(
                     tuple(w.value.sharding.spec), ("batch", "model")
                 )
-            if "ffw_linearl" in w.path:
+            if "ffw_linear" in w.path:
                 self.assertEqual(
                     tuple(w.value.sharding.spec), ("model", "batch")
                 )
+
+    def test_distribution_with_lora(self):
+        if keras.backend.backend() != "jax":
+            return
```

Comment:
I think you can do `self.skipTest("This tests requires jax backend.")` will show up when running a test suite better.

Reply:
Just fixed this up directly as I'm pulling this in.